Visualisation.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+import matplotlib.pyplot as plt 
+import seaborn as sns
+import numpy as np
+
+df = pd.read_table('diffexpr_data.tsv.gz')
```

Comment:
Не надо называть переменную df, если ты выкладываешь это код)
```suggestion
diffexpr_data = pd.read_table('diffexpr_data.tsv.gz')
```

Reply:
А почему...

Reply:
Ну тип не понятно, что в ней лежит. Если уж очень сильно хочется показать, что это именно датафрейм, то можно хотя бы `<what's in the box>_df`